### ISSUE TITLE
New Element Call lobby should show call members

### ISSUE THREAD

**Issue by bblacher** [element-hq/element-web#27030]
#### What would you like to do?
Click on an Element Call powered video room and see who already joined (like it showed before 1.11.58)
#### Why would you like to do it?
Because I want to know who is waiting in an video room so I can evaluate if I should join the room.
#### How would you like to achieve it?
It should work like it did in the old lobby with Element Call or Jitsi.

### Have you considered any alternatives?

Downgrading Element Desktop isn't really an option and I think it's also not advised.

**Comment by t3chguy** [comment]
This would be an issue for https://github.com/element-hq/element-call given the whole thing lives in the iframe, I don't have ability to transfer there

**Comment by bblacher** [comment]
Alright, I opened an issue there: https://github.com/element-hq/element-web/issues/27030

**Comment by AlexFolland** [comment]
It should even show it outside the lobby, like in the room list, just like in Discord.  It's one of the nice things about Discord's voice channels.  That would be outside the Element Call iframe, right?

Would that constitute a separate request?  It seems connected, to me.  Discord's voice room attendance status also propagates to the "server" level (Discord's equivalent to Matrix "spaces") by showing an icon on a server that has anyone in any voice channels.  That would all be outside the Element Call iframe and would be very useful to see.  In my case, I could see when my friends are hanging out together in voice chat while I'm viewing rooms in a different space.

**Comment by bblacher** [comment]
I agree that showing the participants in the room list would be very useful.
